source/opt/optimizer.cpp

```diff
@@ -873,4 +874,9 @@ Optimizer::PassToken CreateDecomposeInitializedVariablesPass() {
       MakeUnique<opt::DecomposeInitializedVariablesPass>());
 }
 
+Optimizer::PassToken CreateSplitInvalidUnreachablePass() {
```

Comment:
Do you add command line options for the passes you are adding?  They are useful and should be added.

Reply:
Created #2574 to track implementing this.